schema.sql

```diff
@@ -0,0 +1,61 @@
+
+-- Table for movies
+CREATE TABLE movies (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  title VARCHAR(255),
+  genre VARCHAR(255),
+  author VARCHAR(255),
+  source VARCHAR(255),
+  label VARCHAR(255),
+  publish_date DATE,
+  archived BOOLEAN
+);
+
+-- Table for sources
+CREATE TABLE sources (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  name VARCHAR(255) 
+)
+-- Create the books table by using the following schema
+
+CREATE TABLE books (
+  id INT PRIMARY KEY,
+  genre VARCHAR(255),
+  author VARCHAR(255),
+  source VARCHAR(255),
+  label_id INT,
+  publish_date DATE,
+  archived BOOLEAN,
+  FOREIGN KEY (label_id) REFERENCES labels(id)
+);
```

Comment:
- [ ] Please instead of VARCHAR(255), the `author`, `label`, and `genre` columns are supposed to be foreign keys which references the id on their respective tables. This is how you ensure the belongs-to relationship between the books and the aforementioned classes.
### HINT 👇 
```sql
CREATE TABLE IF NOT EXISTS book (
  id SERIAL PRIMARY KEY,
  publisher VARCHAR(255) NOT NULL,
  cover_state VARCHAR(255) NOT NULL,
  publish_date DATE NOT NULL,
  archived BOOLEAN NOT NULL DEFAULT FALSE,
  genre_id INT REFERENCES genre(id),
  label_id INT REFERENCES label(id),
  author_id INT REFERENCES author(id)
);
```
- [ ] You will have to create a `Genre` table as it is part of the requirements 👍 
- [ ] Please create the `musicAlbum` table as it is an essential part of the requirements 👍